wsagent/che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConnection.java

```diff
@@ -749,6 +751,20 @@ public Revision commit(CommitParams params) throws GitException {
       commitCommand.setInsertChangeId(isGerritSupportConfigured);
       RevCommit result = commitCommand.call();
 
+      Map<String, List<EditedRegion>> modifiedFiles = new HashMap<>();
+      for (String file : status.getChanged()) {
+        modifiedFiles.put(file, getEditedRegions(file));
+      }
+      // We throw event, because with help file watchers we can not detect time
+      // when operation commit was completed.
```

Comment:
Need to fire this event, because with the help of the file watchers we can not detect JGit's commit operation completion.

Reply:
Done.